<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmNavButton</class>
 <widget class="QWidget" name="frmNavButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widgetNav7" native="true">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>245</y>
     <width>511</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="NavButton" name="navButton71">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>首页</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton72">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>论坛</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton73">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Qt下载</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton74">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>作品展</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton75">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>群组</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton76">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>个人中心</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widgetNav1" native="true">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>120</width>
     <height>133</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="NavButton" name="navButton11">
      <property name="text">
       <string>学生管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton12">
      <property name="text">
       <string>教师管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton13">
      <property name="text">
       <string>成绩管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton14">
      <property name="text">
       <string>记录查询</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widgetNav2" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>11</y>
     <width>120</width>
     <height>133</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="NavButton" name="navButton21">
      <property name="text">
       <string>访客登记</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton22">
      <property name="text">
       <string>记录查询</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton23">
      <property name="text">
       <string>系统设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton24">
      <property name="text">
       <string>系统重启</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widgetNav5" native="true">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>151</y>
     <width>511</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="NavButton" name="navButton51">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>首页</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton52">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>论坛</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton53">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>作品</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton54">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>群组</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton55">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>帮助</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widgetNav6" native="true">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>198</y>
     <width>511</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="NavButton" name="navButton61">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>首页</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton62">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>论坛</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton63">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>作品</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton64">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>群组</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton65">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>帮助</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widgetNav3" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>11</y>
     <width>120</width>
     <height>133</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="NavButton" name="navButton31">
      <property name="text">
       <string>学生管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton32">
      <property name="text">
       <string>教师管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton33">
      <property name="text">
       <string>成绩管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton34">
      <property name="text">
       <string>记录查询</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widgetNav4" native="true">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>11</y>
     <width>120</width>
     <height>133</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="NavButton" name="navButton41">
      <property name="text">
       <string>学生管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton42">
      <property name="text">
       <string>教师管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton43">
      <property name="text">
       <string>成绩管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="NavButton" name="navButton44">
      <property name="text">
       <string>记录查询</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavButton</class>
   <extends>QPushButton</extends>
   <header>navbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
